enos/enos-dev-scenario-pr-replication.hcl

```diff
@@ -6,23 +6,130 @@ scenario "dev_pr_replication" {
     This scenario spins up a two Vault clusters with either an external Consul cluster or
     integrated Raft for storage. The secondary cluster is configured with performance replication
     from the primary cluster. None of our test verification is included in this scenario in order
-    to improve end-to-end speed. If you wish to perform such verification you'll need to a non-dev
-    scenario.
+    to improve end-to-end speed. If you wish to perform such verification you'll need to use a
+    non-dev scenario.
 
     The scenario supports finding and installing any released 'linux/amd64' or 'linux/arm64' Vault
     artifact as long as its version is >= 1.8. You can also use the 'artifact:local' variant to
     build and deploy the current branch!
 
-    In order to execute this scenario you'll need to install the enos CLI:
+    You can use the following command to get a textual outline of the entire
```

Comment:
This is a _lot_ of information. I'm trying figure out whether it all really should be in the scenario outline now or just information that is _specific to each scenario_. It almost feels like half of this information would be more apt to a README than in _every_ scenario description.

Reply:
And I realize that is the opposite of what I've previously said. But seeing it here has me questioning that.